docs/content/policy-reference.md

```diff
@@ -786,6 +786,7 @@ Note that the opa executable will need access to the timezone files in the envir
 | <span class="opa-keep-it-together">``output := crypto.md5(string)``</span> | ``output`` is ``string`` md5 hashed. |
 | <span class="opa-keep-it-together">``output := crypto.sha1(string)``</span> | ``output`` is ``string`` sha1 hashed. |
 | <span class="opa-keep-it-together">``output := crypto.sha256(string)``</span> | ``output`` is ``string`` sha256 hashed. |
+| <span class="opa-keep-it-together">``output := crypto.x509.parse_certificate_request(string)``</span> | ``output`` is a X.509 PKCS #10 certificate represented as JSON objects. |
```

Comment:
nit: Lets move this up next to the `crypto.x509.parse_certificates` definition to group the `crypto.x509` ones together.

Reply:
Done